src/txmempool.h

```diff
@@ -534,6 +534,10 @@ class CTxMemPool
     const setEntries & GetMemPoolParents(txiter entry) const EXCLUSIVE_LOCKS_REQUIRED(cs);
     const setEntries & GetMemPoolChildren(txiter entry) const EXCLUSIVE_LOCKS_REQUIRED(cs);
     uint64_t CalculateDescendantMaximum(txiter entry) const EXCLUSIVE_LOCKS_REQUIRED(cs);
+
+    // track locally submitted transactions & periodically retry initial broadcast
+    std::set<uint256> m_unbroadcast_txids;
```

Comment:
Have you weighted implications of wtxid-relay (#18044) ? I think you will need to track both txid/wtxid for backward-compability with non-upgraded nodes.

Reply:
ah, good point. will think this through. thanks!

Reply:
from https://github.com/bitcoin/bitcoin/pull/18038#issuecomment-606190183

RE: wtxid - there is some additional code needed to support unbroadcast set & wtxid relay. I'll work on the patch & depending on timing around which PR is ready for merge first, include the patch here or offer it on the other PR. Thanks again for pointing that out :)